ensemble/ctf/utils.py

```diff
@@ -1,15 +1,17 @@
-from __future__ import unicode_literals
+from __future__ import division, unicode_literals
+
 import json
 
 import numpy as np
+import six
 
 from .transfer_function import TransferFunction
 
 
 def build_screen_to_function(component):
     def func(pos):
         bounds = component.bounds
-        return tuple([clip_to_unit(z / size) for z, size in zip(pos, bounds)])
+        return tuple([clip_to_unit(z // size) for z, size in zip(pos, bounds)])
```

Comment:
One more division that I am not sure of. @jwiggins do you know off the top of your head if `component.bounds` is always an int or if it is a float?

Reply:
Wrapping the answer in `clip_to_unit` seems to imply a value between 0 and 1.

Reply:
Yup. I've reverted the floor division to normal division.